plugins/github-deployments/src/components/GithubDeploymentsTable/columns.tsx

```diff
@@ -0,0 +1,90 @@
+/*
+ * Copyright 2021 Spotify AB
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+import React from 'react';
+import {
+  StatusPending,
+  StatusRunning,
+  StatusOK,
+  TableColumn,
+  StatusAborted,
+  StatusError,
+} from '@backstage/core';
+import { GithubDeployment } from '../../api';
+import { DateTime } from 'luxon';
+import { Box, Typography, Link } from '@material-ui/core';
+
+const statusIndicator = (value: string): JSX.Element => {
```

Comment:
Random thought: This could have been an exported component instead, `<GithubStateIndicator state={row.state} />` (they seem to use the term state rather than status in the API). Then people could use that in their custom columns.

Reply:
Good idea, I've updated it to be a component + renamed status to state.